test/integration/workers/atlas_worker.js

```diff
@@ -45,17 +47,27 @@ describe('Atlas worker - integration', () => {
     info: () => {},
     error: () => {}
   };
-  let mockStrategy;
+  let mockChallengeStrategy;
+  let mockTransferStrategy;
 
-  const createMockStrategy = () => {
-    mockStrategy = {
-      workerInterval: 5,
+  const createMockChallengeStrategy = () => {
+    mockChallengeStrategy = {
       retryTimeout: 5,
       shouldFetchBundle: sinon.stub().resolves(true),
-      shouldResolveChallenge: sinon.stub().resolves(true),
-      afterChallengeResolution: sinon.stub()
+      shouldResolve: sinon.stub().resolves(true),
+      afterResolution: sinon.stub()
     };
-    mockStrategy.__proto__ = AtlasChallengeParticipationStrategy.prototype;
+    mockChallengeStrategy.__proto__ = AtlasParticipationStrategy.prototype;
+  };
+
+  const createMockTransferStrategy = () => {
```

Comment:
This is not needed, as `mockTransferStrategy` is not used in the Atlas worker.

However, I suggest creating an integration test similar to this one for transfers. Either in this file or in separate one.